tsconfig.json

```diff
@@ -4,6 +4,9 @@
     "src"
   ],
   "compilerOptions": {
+    "jsx": "react",
+    "jsx": "react",
+    "jsx": "react",
```

Comment:
The `jsx` option is declared multiple times. You should only have one `jsx` option set to `react`. Remove the duplicate entries to avoid configuration conflicts.